src/components/SignInButton.jsx

```diff
@@ -0,0 +1,27 @@
+import UserContext from '../context/UserContext';
+import SignedInMsg from './SignedInMsg';
+import {useContext} from 'react'
+
+const SignInButton = () =>{
+    const { loggedInUser, setLoggedInUser, signedIn,setSignedIn } = useContext(UserContext)
+
+    const handleSignIn = () => {
+        if (signedIn === false) {
+            setLoggedInUser(loggedInUser)
+			setSignedIn(!signedIn)
+		} else {
+			setSignedIn(!signedIn)
+            setLoggedInUser(loggedInUser)
+		}
+        console.log(loggedInUser)
```

Comment:
Make sure to remove stray console logs. Also - just a tip, but if you want the logged in user to persist on refresh, you can use your local storage to store the fact you are logged in :)